pkg/config/config.go

```diff
@@ -543,10 +544,52 @@ func SHA256(text string) string {
 	return hex.EncodeToString(algorithm.Sum(nil))
 }
 
+// parseCherryPicksToExternalRefs parses cherry-pick entries and converts them
+// to SPDX ExternalRef entries with category "OTHER" and type "cherry-pick".
+// The format expected is: [branch/]commit-id: comment
+// Each cherry-pick is represented as an ExternalRef with the locator being the
+// branch/commit format
+func parseCherryPicksToExternalRefs(cherryPicks string) []spdx.ExternalRef {
+	lines := strings.Split(cherryPicks, "\n")
+	refs := make([]spdx.ExternalRef, 0, len(lines))
+	for _, line := range lines {
+		line = strings.Split(line, "#")[0]
```

Comment:
Not sure I understand this split.  Are you looking for comments here?

Reply:
My thinking was: certain lines can have inline comments, right? Like in `master/b67d508a93bf1ba231c18dce3894cfee25c16e0d: some description  # This is temporary`. If we don't support this, then I guess we can drop it.